src/pages/saksbehandling/steg/formue/FormueComponents.tsx

```diff
@@ -0,0 +1,31 @@
+import { Input } from 'nav-frontend-skjema';
+import { Element, Undertittel } from 'nav-frontend-typografi';
+import React from 'react';
+
+export const FormueInput = (props: {
+    className: string;
+    tittel: string;
+    inputName: string;
+    defaultValue: number;
+    onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+    feil: string | undefined;
+}) => (
+    <>
+        <h3> {props.tittel} </h3>
+        <Input
+            className={props.className}
+            name={props.inputName}
+            type="number"
+            feil={props.feil}
+            defaultValue={props.defaultValue}
+            onChange={props.onChange}
+        />
+    </>
+);
```

Comment:
Input-feltet trenger en label. F.eks:
```suggestion
export const FormueInput = (props: {
    id: string;
    className: string;
    tittel: string;
    inputName: string;
    defaultValue: number;
    onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
    feil: string | undefined;
}) => (
    <>
        <Label htmlFor={props.id}>
            <h3>{props.tittel}</h3>
        </Label>
        <Input
            id={props.id}
            className={props.className}
            name={props.inputName}
            type="number"
            feil={props.feil}
            defaultValue={props.defaultValue}
            onChange={props.onChange}
        />
    </>
);
```

Burde kanskje vurdere å bruke noe annet enn `h3` også; gjerne noe fra typografi-pakken.

Reply:
alright!